### ISSUE TITLE
ava.lang.RuntimeException: Unable to stop service org.linphone.LinphoneService@a61658b: java.lang.NullPointerException: Attempt to invoke virtual method 'void org.linphone.notifications.NotificationsManager.destroy()' on a null object reference

### ISSUE THREAD

**Issue by brianjmurrell** [BelledonneCommunications/linphone-android#553]
Observed in the logcat for a feature/release-4.1 build:

```
04-05 12:52:43.320 18191 18205 I ActivityManager: Start proc 14597:org.linphone.debug/u0a194 for service org.linphone.debug/org.linphone.LinphoneService
04-05 12:52:43.407 18191 29238 W ActivityManager: Stopping service due to app idle: u0a194 -1h21m33s8ms org.linphone.debug/org.linphone.LinphoneService
04-05 12:52:43.666 14597 14597 W FactoryImpl: Unable to load optional library ffmpeg-linphone: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/org.linphone.debug-7yRj2SbAHarS84OEZF5Gtw==/base.apk"],nativeLibraryDirectories=[/data/app/org.linphone.debug-7yRj2SbAHarS84OEZF5Gtw==/lib/arm, /data/app/org.linphone.debug-7yRj2SbAHarS84OEZF5Gtw==/base.apk!/lib/armeabi-v7a, /system/lib, /system/vendor/lib]]] couldn't find "libffmpeg-linphone.so"
04-05 12:52:43.989 14597 14597 I Linphone: Using (r/w) config information from /data/user/0/org.linphone.debug/files/.linphonerc
04-05 12:52:43.997 14597 14597 I Linphone:  ==== Phone information dump ====
04-05 12:52:43.998 14597 14597 I Linphone: DEVICE=hammerhead
04-05 12:52:43.998 14597 14597 I Linphone: MODEL=Nexus 5
04-05 12:52:43.998 14597 14597 I Linphone: MANUFACTURER=LGE
04-05 12:52:43.998 14597 14597 I Linphone: SDK=27
04-05 12:52:43.998 14597 14597 I Linphone: Supported ABIs=armeabi-v7a, armeabi, 
04-05 12:52:43.999 14597 14597 I Linphone: [Service] Linphone version is 4.1-debug (4117)
04-05 12:52:44.000 14597 14597 I Linphone: Using (r/w) config information from /data/user/0/org.linphone.debug/files/.linphonerc
04-05 12:52:44.019 14597 14597 E AndroidRuntime: Process: org.linphone.debug, PID: 14597
04-05 12:52:44.019 14597 14597 E AndroidRuntime: java.lang.RuntimeException: Unable to stop service org.linphone.LinphoneService@a61658b: java.lang.NullPointerException: Attempt to invoke virtual method 'void org.linphone.notifications.NotificationsManager.destroy()' on a null object reference
04-05 12:52:44.019 14597 14597 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void org.linphone.notifications.NotificationsManager.destroy()' on a null object reference
04-05 12:52:44.019 14597 14597 E AndroidRuntime:        at org.linphone.LinphoneService.onDestroy(LinphoneService.java:407)
04-05 12:52:44.043 18191 18501 I ActivityManager: Process org.linphone.debug (pid 14597) has died: fore SVC 
04-05 13:09:28.569 18191 18205 I ActivityManager: Start proc 15521:org.linphone.debug/u0a194 for broadcast org.linphone.debug/com.google.firebase.iid.FirebaseInstanceIdReceiver
04-05 13:09:28.871 15521 15521 W FactoryImpl: Unable to load optional library ffmpeg-linphone: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/org.linphone.debug-7yRj2SbAHarS84OEZF5Gtw==/base.apk"],nativeLibraryDirectories=[/data/app/org.linphone.debug-7yRj2SbAHarS84OEZF5Gtw==/lib/arm, /data/app/org.linphone.debug-7yRj2SbAHarS84OEZF5Gtw==/base.apk!/lib/armeabi-v7a, /system/lib, /system/vendor/lib]]] couldn't find "libffmpeg-linphone.so"
04-05 13:09:29.027 15521 15521 I Linphone: Using (r/w) config information from /data/user/0/org.linphone.debug/files/.linphonerc
04-05 13:09:29.032 15521 15521 I Linphone:  ==== Phone information dump ====
04-05 13:09:29.033 15521 15521 I Linphone: DEVICE=hammerhead
04-05 13:09:29.033 15521 15521 I Linphone: MODEL=Nexus 5
04-05 13:09:29.033 15521 15521 I Linphone: MANUFACTURER=LGE
```

**Comment by Viish** [comment]
This is indeed a crash but it only happens if there is another crash just before.
Do you have another crash in the adb logcat output ?

**Comment by Viish** [comment]
Fixed in b52bd60ca30dad485e6c6d32dd04611a62f46fe1
